### PR TITLE
Add script to clean simulators results

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: clean_simulation_results.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+if runAll:
+    delete_simulation_results(dry_run, "LoadFlow", constant.DELETE_STUDY_LOADFLOW_RESULTS, constant.GET_LOADFLOW_RESULTS_COUNT)
+    delete_simulation_results(dry_run, "Dynamic simulation", constant.DELETE_STUDY_DYNAMIC_SIMULATION_RESULTS, constant.GET_DYNAMIC_SIMULATION_RESULTS_COUNT)
+    delete_simulation_results(dry_run, "Security analysis", constant.DELETE_STUDY_SECURITY_ANALYSIS_RESULTS, constant.GET_SECURITY_ANALYSIS_RESULTS_COUNT)
+    delete_simulation_results(dry_run, "Sensitivity analysis", constant.DELETE_STUDY_SENSITIVITY_ANALYSIS_RESULTS, constant.GET_SENSITIVITY_ANALYSIS_RESULTS_COUNT)
+    delete_simulation_results(dry_run, "Shortcircuit", constant.DELETE_STUDY_SHORTCIRCUIT_RESULTS, constant.GET_SHORTCIRCUIT_RESULTS_COUNT) 
+    delete_simulation_results(dry_run, "Voltage init", constant.DELETE_STUDY_VOLTAGE_INIT_RESULTS, constant.GET_VOLTAGE_INIT_RESULTS_COUNT)
+else: 
+    if loadflow:
```
Comment (sBouzols) [inline clean_simulation_results.py:60]: avoid code duplication using if loadflow || runAll for example

File: clean_simulation_results.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+parser = argparse.ArgumentParser(description='Send requests to the gridsuite services to delete simulation results', )
+
+parser.add_argument("--dry-run", help="test mode (default) will not execute any modification request",
+                    action="store_true")
+parser.add_argument("-lf", "--loadflow", help="delete all loadflow results",
+                    action="store_true")
+parser.add_argument("-ds", "--dynamicsimulation", help="delete all dynamic simulation results",
+                    action="store_true")
+parser.add_argument("-su", "--security", help="delete all security analysis results",
```
Comment (sBouzols) [inline clean_simulation_results.py:25]: sa

File: clean_simulation_results.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import argparse
```
Comment (sBouzols) [inline clean_simulation_results.py:1]: License @author

File: clean_simulation_results.py (hunk 1 above)
Comment (Meklo) [reply clean_simulation_results.py]: Done

File: clean_simulation_results.py (hunk 2 above)
Comment (Meklo) [reply clean_simulation_results.py]: Done

File: clean_simulation_results.py (hunk 3 above)
Comment (Meklo) [reply clean_simulation_results.py]: Done

File: constant.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# ENV FOLDERS
```
Comment (sBouzols) [inline constant.py:1]: It's a Python script as well then Add a License header as well + @author

Comment (Meklo) [reply constant.py:1]: Done

Comment (Meklo) [reply constant.py:12]: Done

File: functions/clean_computation_results.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+#
+
+import requests
+import constant
+
+#
+# @author Hugo Marcellin <hugo.marcelin at rte-france.com>
+#
+
+def delete_computation_results(dry_run, computationType):
```
[the remark this answers is not in the document]
Comment (Meklo) [reply functions/clean_computation_results.py]: Done

**Review by SlimaneAmar — APPROVED** [review]

**Review by sBouzols — APPROVED** [review]

**Pull request merged**